### PR TITLE
Doc: Managed service account

### PR DESCRIPTION
**Pull request by yue9944882**
> Signed-off-by: yue9944882 <291271447@qq.com>

### CODE REVIEW

File: content/en/getting-started/integration/managed-serviceaccount.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+<!-- spellchecker-disable -->
+
+{{< toc >}}
+
+<!-- spellchecker-enable -->
+
+[Managed Service Account](https://github.com/open-cluster-management-io/managed-serviceaccount)
+is an OCM addon enabling a hub cluster admin to manage [service account](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/)
+across multiple clusters on ease. By controlling the creation and removal of
+the service account, the addon agent will be projecting and rotation the
```
Comment (qiujian16) [inline content/en/getting-started/integration/managed-serviceaccount.md:15]: s/be projecting and rotation/project and rotate

File: content/en/getting-started/integration/managed-serviceaccount.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+[Managed Service Account](https://github.com/open-cluster-management-io/managed-serviceaccount)
+is an OCM addon enabling a hub cluster admin to manage [service account](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/)
+across multiple clusters on ease. By controlling the creation and removal of
+the service account, the addon agent will be projecting and rotation the 
+corresponding token back to the hub cluster which is very useful for the Kube
+API client from the hub cluster to request against the managed clusters.
+
+## Background
+
+Normally there are two major approach for a Kube API client to authenticate and
```
Comment (qiujian16) [inline content/en/getting-started/integration/managed-serviceaccount.md:21]: s/approach/approaches

File: content/en/getting-started/integration/managed-serviceaccount.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+- Service account bearer token
+  
+The service account token will be automatically and persisted as a secret 
+resource inside the hosting Kubernetes clusters upon creation, which is commonly
+used for the ["in-cluster"](https://github.com/kubernetes/client-go/tree/master/examples/in-cluster-client-configuration)
+client. However, in terms of OCM, the hub cluster is completely an external 
+system to the managed clusters, so we will need a local agent in each managed 
+cluster to reflect the tokens consistently to the hub cluster so that the
+Kube API client from hub cluster can "push" the requests directly against the
+managed cluster. FWIW, by delegating the multi-cluster service account
```
Comment (qiujian16) [inline content/en/getting-started/integration/managed-serviceaccount.md:34]: maybe not FWIW here :)

File: content/en/getting-started/integration/managed-serviceaccount.md (hunk 1 above)
Comment (yue9944882) [reply content/en/getting-started/integration/managed-serviceaccount.md]: updated

File: content/en/getting-started/integration/managed-serviceaccount.md (hunk 2 above)
Comment (yue9944882) [reply content/en/getting-started/integration/managed-serviceaccount.md]: udpated

File: content/en/getting-started/integration/managed-serviceaccount.md (hunk 3 above)
Comment (yue9944882) [reply content/en/getting-started/integration/managed-serviceaccount.md]: removed FWIW

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**